### DISCUSSION TITLE
Session start error: SEB failed to start a new session!

### DISCUSSION THREAD

**Discussion by rouhbl** [SafeExamBrowser/seb-win-refactoring#426] [Q&A]
Good day SEB community,

In the last few months I have been running into this error:

```
Session Start Error:

SEB failed to start a new session! Please Consult the log files for more information. 
(logfile added)
```

we use SEB 3.3.2 and we have been running into this issue since we started using it, the most annoying part though is that we cant seem to consistently reproduce the issue, one day we will can turn on all the machines and launch SEB without issue, then the next day 70% of those same machines can have the above said error.

From the SEB FAQ I found this topic which in my eyes might have something to do with this. but even then I cant seem to reproduce the issues (shutting down normally, force shut down, normal shut down of SEB, forced shut down with SEB running)

none of the above methods seemed to reproduce the issues I see when starting up a machine with SEB at seemingly random intervals.

Restarting seems to resolve the issue sometimes. but not always.

Thank you for your time in advance

[2022-06-17_09h05m44s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/8925853/2022-06-17_09h05m44s_Runtime.log)

**Comment by dbuechel** [comment]
The SEB configuration you're trying to use has the SEB Service policy set to mandatory (see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane). Thus, the SEB Service needs to be running in order to start SEB, but that does not appear to be the case as you explained and as also can be seen in the log file:

```
2022-06-17 09:05:45.582 [04] - ERROR: The service is mandatory but no connection could be established!
```

There hence must be an issue on your systems which prevents the SEB Service from running correctly after system boot. You can either try to fix that issue or then manually ensure that the SEB Service is running (i.e. restart it when encountering this issue).
